src/reactionsystem.jl

```diff
@@ -763,13 +796,59 @@ function get_indep_sts(rs::ReactionSystem, remove_conserved = false)
     indepsts, filter(isspecies, indepsts)
 end
 
+"""
+    parameters_toplevel(network; initial_parameters = false)
+
+Returns the parameters of the top-level system only. Equivalent to `parameters`, but ignores
+parameters of sub-systems.
+
+WARNING:
+- This function may be removed in non-breaking changes.
```

Comment:
Right now I marked this as something that might be removed in non-breaking releases. Which might not be ok since we briefly mentions it in the docs.